java/ql/src/semmle/code/java/security/ResponseSplitting.qll

```diff
@@ -5,41 +5,30 @@ import semmle.code.java.dataflow.DataFlow
 import semmle.code.java.dataflow.FlowSources
 import semmle.code.java.frameworks.Servlets
 import semmle.code.java.frameworks.JaxWS
+private import semmle.code.java.dataflow.ExternalFlow
 
 /** A sink that is vulnerable to an HTTP header splitting attack. */
-abstract class HeaderSplittingSink extends DataFlow::Node { }
+class HeaderSplittingSink extends DataFlow::Node {
```

Comment:
We can't just remove the abstract class. That's an extension point for anyone using the `Customizations.qll` extension mechanism. Instead we'll need to subclass it with a `DefaultHeaderSplittingSink extends HeaderSplittingSink`.